src/config/modelProviders/groq.ts

```diff
@@ -3,6 +3,27 @@ import { ModelProviderCard } from '@/types/llm';
 // ref https://console.groq.com/docs/models
 const Groq: ModelProviderCard = {
   chatModels: [
+    {
+      displayName: 'LLaMA3.1 405B',
+      enabled: true,
+      functionCall: true,
+      id: 'llama-3.1-405b-reasoning',
+      tokens: 131_072,
```

Comment:
During the preview release, all 3.1 models had max_tokens of 8k, and 405b had 16k input tokens. So I think the `tokens` of these models also need to be modified.